### PR TITLE
FOGL-1859: SP-c++/plugins - modify hardcoded configuration.

### PR DESCRIPTION
**Pull request by sstefano69**

### CODE REVIEW

File: C/plugins/north/PI_Server_V2/plugin.cpp [hunk 1]
```diff
@@ -41,6 +41,11 @@ using namespace rapidjson;
 				"\"description\": \"The producer token that represents this FogLAMP stream\", " \
 				"\"type\": \"string\", \"default\": \"omf_north_0001\", " \
 				"\"order\": \"2\" }, " \
+			"\"source\": {" \
```
Comment (MarkRiddoch) [inline C/plugins/north/PI_Server_V2/plugin.cpp]: This should be an enumeration rather than a string

Comment (sstefano69) [reply C/plugins/north/PI_Server_V2/plugin.cpp]: you are right, fixing.

**Comment by sstefano69** [comment]
@MarkRiddoch -
the default  for the 'compression' :
* is true in the Python
* is false in the c++

is the c++ right ?

**Comment by MarkRiddoch** [comment]
I suspect having compression on by default is probably a good thing. The ability to turn it off if there is a problem is probably wise to have however.

File: C/plugins/north/PI_Server_V2/plugin.cpp (hunk 1 above)
Comment (sstefano69) [reply C/plugins/north/PI_Server_V2/plugin.cpp]: fixed

**Pull request merged**